src/web-server/html-formatter.js

```diff
@@ -57,72 +57,89 @@ const formatRow = (
+                        <body><h1>Index of ${dirPath}</h1>
+                        <table><tr><th><img src="${iconBlank}" alt="[ICO]"></th>
+                        <th><b>Name</b></th><th><b>Last modified</b></th>
+                        <th><b>Size</b></th><th><b>Description</b></th></tr>
+                        <tr><th colspan="5"><hr></th></tr>
+                        <tr><td valign="top"><img src="${iconBack}" alt="[DIR]"></td>
+                        <td><a href="/www${parent}">Parent Directory</a></td><td>&nbsp;</td>
+                        <td align="right">  - </td><td>&nbsp;</td></tr>`;
 
-    const footer = `<tr><th colspan="5"><hr></th></tr></table>${footerClose}`;
+        const footer = `<tr><th colspan="5"><hr></th></tr></table>${footerClose}`;
 
-    const rows = entries
-        .map(entry => {
-            const ext = path.extname(entry.name);
-            const href = '/www' + path.join(dirPath, entry.name);
-            let icon;
-            let alt;
+        const rows = entries
+            .map(entry => {
+                const ext = path.extname(entry.name);
+                const href = '/www' + path.join(dirPath, entry.name);
+                let icon;
+                let alt;
 
-            if (entry.type === 'DIRECTORY') {
-                icon = iconFolder;
-                alt = '[DIR]';
-            } else {
-                if (isImage(ext)) {
-                    icon = iconImage;
-                    alt = '[IMG]';
-                } else if (isMedia(ext)) {
-                    icon = iconMovie;
-                    alt = '[MOV]';
+                if (entry.type === 'DIRECTORY') {
+                    icon = iconFolder;
+                    alt = '[DIR]';
                 } else {
-                    icon = iconText;
-                    alt = '[TXT]';
+                    if (isImage(ext)) {
+                        icon = iconImage;
+                        alt = '[IMG]';
+                    } else if (isMedia(ext)) {
+                        icon = iconMovie;
+                        alt = '[MOV]';
+                    } else {
+                        icon = iconText;
+                        alt = '[TXT]';
+                    }
                 }
-            }
 
-            return formatRow(
-                icon,
-                alt,
-                href,
-                entry.name,
-                entry.mtime,
-                entry.size
-            );
-        })
-        .join('\n');
+                return formatRow(
+                    icon,
+                    alt,
+                    href,
+                    entry.name,
+                    entry.mtime,
+                    entry.size
+                );
+            })
+            .join('\n');
+
+        return {
+            type: 'text/html',
+            status: 200,
+            body: header + rows + footer,
+        };
+    },
 
-    return header + rows + footer;
+    formatFile: ({ path, content }) => {
```

Comment:
```js
formatFile: ({ path, content }) => {
    body: ...
}
```

Reply:
```js
{
    body: ...
}
```

 will be considered a function body.

The proper way of doing it
```js
Object.create({ body: ... })
```
I just feel like what I wrote looks cleaner.

Same for others.

Reply:
OK, sounds good.